Taskfile.yaml

```diff
@@ -122,12 +123,12 @@ tasks:
     desc: Run all unit tests
     vars:
       TEST_PKGS:
-          sh: "go list ./... | grep -v {{ .OWNER }}/{{ .PROJECT }}/test | tr '\n' ' '"
+          sh: "python ./scripts/list_units.py anchore binny"
```

Comment:
Since we have binny, can we just have it download `grep` (and maybe `sh`, `curl`, and other such missing shell script utilities) instead of making the python scripts?

Reply:
My concern is that the utilities won't behave the same even if we make them available. For example `ls` lists files in PowerShell, but the output looks different than on macOS or Linux. We also see elsewhere in this PR that `mkdir -p` doesn't work on Windows, even though there _is_ a mkdir on PATH in PowerShell.

In other words, if we keep using a command shell as our scripting environment, we have to worry about the compatibility of every executable the shell invokes. If we use standard library Python, we only have to worry about the cross-platform behavior of Python itself.